src/UnitTests/Scanning/BlockConflictResolverTests.cs

```diff
@@ -74,6 +76,38 @@ private void When_DisassembleProcedure()
                 mem.EndAddress);
         }
 
+        private RtlBlock Given_Block(uint uAddr)
+        {
+            var b = new RtlBlock(Address.Ptr32(uAddr), $"l{uAddr:X8}");
+            return b;
+        }
+
+        private void Given_Instrs(RtlBlock block, Action<RtlEmitter> b)
+        {
+            var instrs = new List<RtlInstruction>();
+            var trace = new RtlEmitter(instrs);
+            b(trace);
+            var rtlc = InstrClass.None;
+            if (instrs.Count == 1)
```

Comment:
Would it make more "realistic" to have `rtlc` be `InstrClass.Linear`?

Reply:
Done